### PR TITLE
Upgrade grafana chart

### PR DESCRIPTION
**Pull request by skhalash**
**Description**

Changes proposed in this pull request:

- Upgrade Grafana chart to v6.1.9 of the upstream

**Related issue(s)**

### CODE REVIEW

File: resources/monitoring/charts/grafana/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 apiVersion: v1
 name: grafana
-version: 5.2.1
+version: 6.1.9
+appVersion: 7.3.3
```
Comment (suleymanakbas91) [inline resources/monitoring/charts/grafana/Chart.yaml:4]: We decided to delete `appVersion` field as it is mostly forgotten to be maintained: https://github.com/kyma-project/kyma/commit/a83c6821641471141dfd547cebcec0abfbb23530. Could you please delete it from all the subcharts?

File: resources/monitoring/charts/grafana/values.yaml [hunk 2]
```diff
@@ -63,6 +67,13 @@ image:
   # pullSecrets:
   #   - myRegistrKeySecretName
 
+testFramework:
```
Comment (suleymanakbas91) [inline resources/monitoring/charts/grafana/values.yaml:70]: You can delete this part as we are not using the Grafana test framework.

**Review by suleymanakbas91 — CHANGES_REQUESTED** [review]

File: resources/monitoring/charts/grafana/Chart.yaml (hunk 1 above)
Comment (skhalash) [reply resources/monitoring/charts/grafana/Chart.yaml]: Done

File: resources/monitoring/charts/grafana/values.yaml (hunk 2 above)
Comment (skhalash) [reply resources/monitoring/charts/grafana/values.yaml:70]: Done

**Review by suleymanakbas91 — APPROVED** [review]

**Pull request merged**